challenge/resolver/prober.go

```diff
@@ -149,14 +149,14 @@ func parallelSolve(authSolvers []*selectedAuthSolver, failures obtainError) {
 	// Finally solve all challenges for real
 	for _, authSolver := range authSolvers {
 		authz := authSolver.authz
-		if failures[authz.Identifier.Value] != nil {
+		if failures[challenge.GetTargetedDomain(authz)] != nil {
```

Comment:
Why not just pull `domain := challenge.GetTargetedDomain(authz)` at the top of the loop?

Reply:
because it's only use one time inside the loops.